airbyte-webapp/src/views/layout/SideBar/SideBar.module.scss

```diff
@@ -1,6 +1,35 @@
 @use "../../../scss/colors";
 @use "../../../scss/variables";
 
+.nav {
+  z-index: 9999;
```

Comment:
we have a `z-indices.scss` file where we are keeping track of these globally.  Please add a new sidebar variable and import it here.